src/util/system.cpp

```diff
@@ -1128,10 +1128,22 @@ fs::path GetSpecialFolderPath(int nFolder, bool fCreate)
 void runCommand(const std::string& strCommand)
 {
     if (strCommand.empty()) return;
+    int nErr = 0;
 #ifndef WIN32
-    int nErr = ::system(strCommand.c_str());
+    nErr = ::system(strCommand.c_str());
 #else
-    int nErr = ::_wsystem(std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>,wchar_t>().from_bytes(strCommand).c_str());
+    STARTUPINFOA info;
+    PROCESS_INFORMATION processInfo;
+    ZeroMemory(&info, sizeof(info));
+    info.cb = sizeof(info);
+    ZeroMemory(&processInfo, sizeof(processInfo));
+    if (CreateProcessA(NULL, const_cast<char *>(strCommand.c_str()), NULL, NULL, FALSE, 0x08000000, NULL, NULL, &info, &processInfo)) {
+        WaitForSingleObject(processInfo.hProcess, INFINITE);
+        CloseHandle(processInfo.hProcess);
+        CloseHandle(processInfo.hThread);
+    } else {
+        nErr = GetLastError();
+    }
 #endif
     if (nErr)
         LogPrintf("runCommand error: system(%s) returned %d\n", strCommand, nErr);
```

Comment:
This message is explicit about the call being `system`, so should probably be made ambiguous